src/constructLayoutEngine.js

```diff
@@ -150,8 +200,8 @@ function getContainerEl(parentContainer, route) {
  * @param {import('./constructRoutes').Route} route
  * @returns {HTMLElement}
  */
-function getApplicationElement(route) {
-  const htmlId = `single-spa-application:${route.name}`;
+function getApplicationElement(name) {
+  const htmlId = `single-spa-application:${name}`;
```

Comment:
Small question. Are we going to allow id's that aren't generated or only generated ids? As far as I can tell this only supports generated ids and I'm assuming we're going to keep it that way.

Reply:
🤷‍♀️ for now it's the generated ids, I am not opposed to changing it in the future. I don't think people care about the ids as much as about the layout, though?